bundles/hash.rst

```diff
@@ -0,0 +1,44 @@
+HashBundle
+==========
+
+The SuluHashBundle adds an easy to use solution for protecting entities and
+documents from being overridden by accident.
+
+Serialization
+-------------
+
+The `HashSerializeEventSubscriber` is the most important part of the
+HashBundle, since it is its job to add the hash of the object to the serialized
```

Comment:
"HashBundle. The subscriber's job is to add ..."

In general, try to keep sentences short. Use simple grammar. That's much easier to read. :)